crates/formats/Cargo.toml

```diff
@@ -22,6 +22,7 @@ rayon.workspace = true
 rsa = "0.9"
 thiserror.workspace = true
 widestring = "1"
+utf16string = "0.2" # Using both this and widestring is eh, but widestring doesn't support byteorder
```

Comment:
Are we able to get rid of one of them?

Reply:
I thought about removing `widestring`, but `utf16string` doesn't support string slices of potentially invalid utf16. On the other hand, `widestring` doesn't support non-native endianness which is required for param files. Do we really need support for slices with potentially invalid utf16 characters?

Reply:
I suppose that really depends on how the game behaves in the presence of malformed utf16

Reply:
Given that the game will either use `std::wstring` or `wchar_t*` for wide strings, neither of which disallow malformed utf16, we should accept it whenever possible. Keeping both crates isn't ideal, but we have to unless we accept to either
- Fail parsing for files the game would consider valid (granted, where strings are expected in vanilla game files, we would expect them to be properly encoded);
- Drop support for big-endian params with wide string names;
- Write our own widestring crate with support for both malformed utf-16 and non-native endianness.

Reply:
I think option 1 makes the most sense until the need arises to go with option 3.